source/server/hot_restart.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+
+package envoy;
+
+option java_outer_classname = "HotRestartProto";
```

Comment:
nit: you can kill this Java stuff unless check format requires it. If so ignore.

Reply:
Done.